data/json/items/gun/9mm.json

```diff
@@ -29,6 +29,13 @@
       [ "BURST", "3 rd.", 3 ],
       [ "AUTO", "auto", 5 ]
     ],
+    "magazines": [
```

Comment:
Starting a world in experimental 4938 produces errors related to ammo/222.json, ammo/545x39.json, ammo/9x18.json, ammo/metal_rail.json, gun/223.json, gun/308.json, and gun/9mm.json.